<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>agregarProducto</class>
 <widget class="QDialog" name="agregarProducto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Producto nuevo</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Agregar Producto</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Precio por unidad</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="sp_caducidad">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2015</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>d/M/yy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rd_vegetales">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Vegetales</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rd_carnes">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>100</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Carnes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sp_cantidad">
   <property name="geometry">
    <rect>
     <x>237</x>
     <y>170</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Caducidad</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rd_frutas">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Frutas</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rd_postres">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>180</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Postres</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rd_granos">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>140</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Granos</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rd_congelados">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>congelados</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cantidad</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_nombreP">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Alimentos</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>Bebidas</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rd_sodas">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Soda</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rd_jugos">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>290</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Jugo</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rd_agua">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>250</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Agua</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rd_alcohol">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>290</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Vino</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="sp_precio">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>106</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>226</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Marca</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_marca">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_color">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>330</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>agregarProducto</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
